storage/innobase/fil/fil0crypt.cc

```diff
@@ -1116,6 +1166,15 @@ bool fil_crypt_exclude_tablespace_from_rotation(fil_space_t *space) {
   return true;
 }
 
+void fil_crypt_readd_space_to_rotation(space_id_t space_id) {
```

Comment:
s/readd/read

suggestion: fil_crypt_read_rotation_status()

Reply:
actually, it is readd_space_to_rotation - the space->exclude_from_rotation is assigned to false in this function. I can change the name of it to something else if it is to weird ...